css/main.css

```diff
@@ -103,10 +103,18 @@ body {
   font-weight: 700;
 }
 
+.img-container {
+  min-width: 200px;
+  min-height: 200px;
+}
```

Comment:
Can't it be done without this image container ??

Reply:
Let me test it quicky.

Reply:
Yes, the issue was solved without this image container, hence removed it.